### ISSUE TITLE
使用时间越长越卡，每天至少退出重新打开一次

### ISSUE THREAD

**Issue by maningphp** [electerm/electerm#2451]
### Electerm Version(Electerm版本)

1.21.14

### Platform detail (平台详情)

win10

### What steps will reproduce the bug?(重新问题的详细步骤)

使用时间越长越卡，每天至少退出重新打开一次。切换标签会卡，显示日志也会卡，哪哪都卡，重开一下就好了。时间一长就会卡

### What should have happened?(期望的结果)

不卡

**Comment by hummingsoul** [comment]
![2022-04-13 16-55-43屏幕截图](https://user-images.githubusercontent.com/26573994/163140391-29261dc4-7216-468a-a1fc-c7f88f20509d.png)
无语啊 本来linux系统 现在不敢开多个标签 搞得都想换回用只带的终端了

**Comment by gwinfo** [comment]
是的。我使用最新版 v1.21.20 也有这问题。包括多个终端批量输入命令时候，批量开关也卡顿。 还有就是刷新重连断线的终端，也特别卡。  应该不是网络问题，因为重新打开就好了。或者，我换用SecureCRT就不存在这些问题。不知道是不是个例。

**Comment by wolfoot** [comment]
我也是遇到这种情况，软件版本 v1.21.57 系统版本:Ubuntu 20.04 用了几个小时就会卡，关掉重开就好了。

**Comment by zxdong262** [comment]
最新版已经优化
